src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_api.py

```diff
@@ -126,6 +126,15 @@ def request_stack(self, py_db, seq, thread_id, fmt=None, timeout=.5):
         else:
             py_db.post_internal_command(internal_get_thread_stack, '*')
 
+    def request_exception_info(self, py_db, request, thread_id):
```

Comment:
This should be `request_exception_info_json` as it only handles the json case.

i.e.: The cases where the postfix is not needed are the ones where the message is received by the proper frontend (xml or json) then passed on through to the api (the same call for both xml and json) and the final dispatch is handled by the current `py_db.cmd_factory` (which is either `NetCommandFactory` or `NetCommandFactoryJson`). On the cases where there's very little reuse of code, covering only one of those is fine, but the method should be postfixed accordingly.